csrc/id_model/id_model.cpp

```diff
@@ -360,9 +362,81 @@ void IdModel::buildExactGraph(const std::vector<Expr*>& exprs) {
   }
 }
 
+namespace {
+
+// Checks if the expression is a trivial operation where an input is simply an
+// output of the transformation. Returns the mapped iter domains if found.
+std::vector<std::vector<Val*>> isTrivialExpr(Expr* expr) {
```

Comment:
Corresponds to:

https://github.com/NVIDIA/Fuser/blob/main/csrc/compute_at_map.cpp#L714-L734